_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -237,7 +237,8 @@ def check_min_allowed_max_date(self, max_date, geo_type, signal_type, report):
                 ValidationFailure("check_min_max_date",
                                   geo_type=geo_type,
                                   signal=signal_type,
```

Comment:
does it make sense to add this here too?
```suggestion
                                  geo_type=geo_type,
                                  signal=signal_type,
                                  date=max_date,
```